aptos-move/framework/aptos-framework/sources/aptos_governance.move

```diff
@@ -274,6 +285,81 @@ module aptos_framework::aptos_governance {
         );
     }
 
+    /// Create a proposal with the backing `stake_pool`.
+  /// @param execution_hash Required. This is the hash of the resolution script. When the proposal is resolved,
+  /// only the exact script with matching hash can be successfully executed.
+    public entry fun create_proposal_generic(
```

Comment:
For future compatibility consideration maybe `create_multi_proposal` is better? My concern is in the future if we want to add new functionailities here we won't be able to use `create_proposal_generic` any more because of the upgrade checks.

Reply:
oooh I see, will update the naming : )